config/routes.rb

```diff
@@ -20,11 +20,10 @@
 
   resources :sessions
 
-  get 'user/index', to: 'users#index'
-  get '/user_todos/:id' => 'users#user_Todos'
-  get '/user/:id'=> 'users#show'
-  put "user_email/:id" => "users#update_Email"
-  put "user_password/:id" => "users#update_Password"
+  get 'user/index' => 'users#index'
```

Comment:
（want）本当は下記が理想

下記を見れば納得すると思います！
https://railsguides.jp/routing.html
```suggestion
  get 'users' => 'users#index'
```

（more want）
`resources`をうまく使ってほしいです（todosと同じ感じで）